utils/ixdtf/src/error.rs

```diff
@@ -46,6 +46,8 @@ pub enum ParseError {
     TimeSecond,
     #[displaydoc("Invalid character while parsing fraction part value.")]
     FractionPart,
+    #[displaydoc("Fraction part value exceeds a representable range.")]
+    InvalidFractionRange,
```

Comment:
"Invalid" is not very descriptive. This is an error, that already tells me that something is not valid.
```suggestion
    ExcessiveSecondPrecision,
```

Reply:
I was double checking this and the error is specific to a Duration hour fraction and minute fraction potentially exceeding the range of `u64::MAX` when calculated. Maybe `DurationFractionExceededRange` (this would then be moved down into the Duration errors).